examples/editor/yarn.lock

```diff
@@ -0,0 +1,5996 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
We don't commit the lockfiles inside examples.

Reply:
We have been for the other ones. Let's tackle removing them if it makes sense in another PR.